src/Stryker.Core/Stryker.Core/DiffProviders/GitDiffProvider.cs

```diff
@@ -39,6 +39,12 @@ public DiffResult ScanDiff()
                 throw new InputException("Could not determine a commit to check for diff. Please check you have provided the correct value for --git-source");
             }
 
+            var testPaths = _options.TestProjects
```

Comment:
In the case of a 'standard' configuration (running from the test project folder) the TestProjects list is empty. So with this change the standard operating mode would no longer work with the since feature.

Reply:
Thank you @rouke-broersma, I added a test and adapted the implementation to also cover this case!